tests/pytests/test_search_params.py

```diff
@@ -58,8 +58,6 @@ def test_param_errors(env):
     env.assertEqual(conn.execute_command('HSET', 'key1', 'foo', 'PARAMS', 'bar', 'PARAMS'), 2)
 
     # Test errors in PARAMS definition: duplicated param, missing param value, wrong count
-    env.expect('FT.SEARCH', 'idx', '*', 'NOCONTENT', 'PARAMS', '4', 'param1', 'val1').error().contains('Bad arguments for PARAMS: Expected an argument')
-    env.expect('FT.SEARCH', 'idx', '*', 'NOCONTENT', 'PARAMS', '2', 'param1').error().contains('Bad arguments for PARAMS: Expected an argument')
```

Comment:
why is it removed?

Reply:
Validations that depend on argument count on the shard are not all good currently due to the arguments we add in the coordinator. This as well will be solved when we do the validations on the coordinator.

Reply:
I'll bring it back for non-cluster case at this point

Reply:
Returned since I moved the new arguments closer to the query